src/main/java/uk/gov/hmcts/ccd/DefaultCacheWarmUpService.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.ccd;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.data.definition.DefaultCaseDefinitionRepository;
+import uk.gov.hmcts.ccd.idam.AuthenticatedUser;
+import uk.gov.hmcts.ccd.idam.IdamHelper;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+import java.util.List;
+
+@Service
+@Qualifier(DefaultCacheWarmUpService.QUALIFIER)
+public class DefaultCacheWarmUpService implements CacheWarmUpService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CachingConfiguration.class);
+    public static final String QUALIFIER = "default";
+    private static final long TWO_SECONDS = 2000l;
```

Comment:
can we have the sleep interval configurable per env please

Reply:
What name would you suggest? Can we go with cacheWarmUpDelay?

Reply:
`cache.warmup.sleep.time` - cacheWarmUpSleepTime ?

Reply:
Cool sounds good to me.